apps/android/app/src/main/java/chat/simplex/app/views/database/DatabaseView.kt

```diff
@@ -196,6 +201,24 @@ fun DatabaseLayout(
         stringResource(R.string.stop_chat_to_enable_database_actions)
       }
     )
+    SectionSpacer()
+    SectionView {
+      SettingsActionItem(
+        Icons.Outlined.DeleteForever,
+        stringResource(R.string.delete_files_and_media),
+        deleteAppFilesAndMedia,
+        textColor = Color.Red,
+        disabled = operationsDisabled || appFilesCountAndSize.value.first == 0
```

Comment:
I think it doesn't have to be disabled when chat is running - at worst a file can be deleted when it's being sent or received, but it doesn't crash backend
```suggestion
        textColor = Color.Red
```

Reply:
I've tested in terminal though, we should test on mobile as well